app/main.py

```diff
@@ -1,3 +1,62 @@
+from __future__ import annotations
+from typing import Any
+
+
 class Distance:
     # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Any) -> Distance:
```

Comment:
and please fix type annotations for func arguments everywhere) 
`Any` its not the best here
try smtn like this `Distance | int | float` etc.